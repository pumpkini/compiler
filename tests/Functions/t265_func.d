
bool k() {
    return true;
}
int h() {
    return 2;
}

double g() {
    return 5.1234;
}

string f(){
    return "f";
}


int main()  {
    Print(f());
    Print(g());
    Print(h());
    Print(k());
}