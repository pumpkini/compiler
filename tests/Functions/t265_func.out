f
5.12340021
2
true
